### ISSUE TITLE
Gatsby error: key.includes is not a function

### ISSUE THREAD

**Issue by hunterbecton** [gatsbyjs/gatsby#10804]
## Description

I'm getting an error: **key.includes is not a function** when running gatsby develop or gatsby build. I haven't changed anything since last night—I just ran gatsby develop this morning and ran into this issue.

### Steps to reproduce

Run gatsby develop or gatsby build from project folder.

Repo: https://github.com/hunterbecton/skillthrive-contentful-auth0/tree/key-bug

### Expected result

Successful development and build status.

### Actual result

Got this error:
```
error key.includes is not a function


  TypeError: key.includes is not a function

  - infer-graphql-type.js:353 _.each
    [skillthrive-contentful-auth0]/[gatsby]/dist/schema/infer-graphql-type.js:35    3:20

  - lodash.js:4887
    [skillthrive-contentful-auth0]/[lodash]/lodash.js:4887:15

  - lodash.js:9344 Function.forEach
    [skillthrive-contentful-auth0]/[lodash]/lodash.js:9344:14

  - infer-graphql-type.js:338 _inferObjectStructureFromNodes
    [skillthrive-contentful-auth0]/[gatsby]/dist/schema/infer-graphql-type.js:33    8:5

  - infer-graphql-type.js:409 inferObjectStructureFromNodes
    [skillthrive-contentful-auth0]/[gatsby]/dist/schema/infer-graphql-type.js:40    9:10

  - build-node-types.js:137 inferFields
    [skillthrive-contentful-auth0]/[gatsby]/dist/schema/build-node-types.js:137:    26

  - build-node-types.js:186 fields
    [skillthrive-contentful-auth0]/[gatsby]/dist/schema/build-node-types.js:186:    19

  - Array.forEach

  - Array.forEach

  - Array.forEach

```

### Environment
```

  System:
    OS: macOS 10.14.2
    CPU: (12) x64 Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 10.14.1 - /usr/local/bin/node
    npm: 6.5.0 - /usr/local/bin/npm
  Browsers:
    Chrome: 71.0.3578.98
    Firefox: 64.0
    Safari: 12.0.2
  npmPackages:
    gatsby: ^2.0.63 => 2.0.63
    gatsby-cli: ^2.4.8 => 2.4.8
    gatsby-image: ^2.0.25 => 2.0.25
    gatsby-plugin-google-fonts: 0.0.4 => 0.0.4
    gatsby-plugin-manifest: ^2.0.7 => 2.0.7
    gatsby-plugin-offline: ^2.0.12 => 2.0.12
    gatsby-plugin-react-helmet: ^3.0.1 => 3.0.1
    gatsby-plugin-sharp: ^2.0.17 => 2.0.17
    gatsby-source-contentful: ^2.0.20 => 2.0.20
    gatsby-source-filesystem: ^2.0.7 => 2.0.7
    gatsby-transformer-remark: ^2.1.15 => 2.1.15
    gatsby-transformer-sharp: ^2.1.9 => 2.1.9
  npmGlobalPackages:
    gatsby-cli: 2.4.6
```

**Comment by sidharthachatterjee** [comment]
Sorry that you're seeing this @hunterbecton 

I tried reproducing this with a sample contentful instance but I wasn't able to. This means the issue is likely specific to your contentful instance (some data causing an edge case that is breaking this?)

Can you maybe try to link to a minimal reproduction of this? Alternatively, you can privately invite me to a repo with access to contentful and I'll be glad to take a look

**Comment by hunterbecton** [comment]
I originally said I didn't change anything, but I remembered that I created two fields in a content type within Contentful. I deleted the field types (one which was an integer and one that was short text) and everything works fine again. So, like you said, it was an issue within my Contentful instance. Not sure what caused the issue exactly, but I'll update the thread in case I figure it out. Thank you!
